src/lib/dom.generated.d.ts

```diff
@@ -2176,6 +2214,7 @@ declare var BhxBrowser: {
     new(): BhxBrowser;
 };
 
+/** The BiquadFilterNode interface represents a simple low-order filter, and is created using the AudioContext.createBiquadFilter() method. It is an AudioNode that can represent different kinds of filters, tone control devices, and graphic equalizers. */
```

Comment:
Wouldn't it improve readability+quality to cut leading `The BiquadFilterNode interface`?